### PR TITLE
Rebase upstream

### PR DESCRIPTION
**Pull request by chaitanyaavi**

### CODE REVIEW

**Review by mayank-avinetworks — APPROVED** [review]

File: examples/cloud/vmware/variables.tf [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  type    = string
+  default = "WRITE_ACCESS"
+}
+variable "vcenter_vcenter_url" {
+  type    = string
+  default = ""
+}
+variable "vcenter_license_tier" {
+  type    = string
+  default = "ENTERPRISE_18"
```
Comment (mayank-avinetworks) [inline examples/cloud/vmware/variables.tf:51]: we need to change this to ENTERPRISE.

If it is 18.2.10 it shd be ent_18
For 20.1.1 it shd be ent

Mayank, Am I correct

Comment (chaitanyaavi) [reply examples/cloud/vmware/variables.tf:51]: @mayank-avinetworks This is just rebase from the upstream we need to change it in upstream First.

Comment (chaitanyaavi) [reply examples/cloud/vmware/variables.tf:51]: 20.1.1 PR to Upstream is not yet done so this is correct.

Comment (mayank-avinetworks) [reply examples/cloud/vmware/variables.tf:51]: Thanks @chaitanyaavi , got it.

**Pull request merged**